hera_mc/tests/test_connections.py

```diff
@@ -40,6 +40,5 @@ def test_commit_part(self):
         print(part)
         self.test_session.add(part)
         self.test_session.commit()
+        #self.assertTrue
```

Comment:
I have no idea on this one  -- not sure when/why/where it happened.  I'll uncomment it for now, but will be fixing/adding tests in the future and will address more then.